code/client/src/Client.cpp

```diff
@@ -0,0 +1,87 @@
+#include <Client.h>
+
+bool Client::updateModule(){
+//    sendRequest();
+    return true;
+}
+
+void Client::sendRequest() {
+
+    yarp::os::Bottle outputBottle;
```

Comment:
It can help if you rename these two Bottles as e.g `request` and `reply`.

The concept of `input` and `output` are a bit obfuscated here.